recipes/r-robustgasp/meta.yaml

```
@@ -0,0 +1,87 @@
+{% set version = '0.6.5' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-robustgasp
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/RobustGaSP_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/RobustGaSP/RobustGaSP_{{ version }}.tar.gz
+  sha256: 0ed297111afd3aaf0d34e8ef5e48e2f5476f6be04a53b0fc2711efd5c707dc9a
+
+build:
+  merge_build_host: True  # [win]
```

Workaround for missing DSO:

```suggestion
  merge_build_host: True  # [win]
  missing_dso_whitelist:
    - '*/R.dll'           # [win]
```

---

Thanks for the prompt response. 
I’d like to know what a DSO is? Would appreciate any info to read on :)